matterbridge.go

```diff
@@ -25,6 +27,7 @@ func NewBridge(name string, config *Config) *Bridge {
 	b.Config = config
 	b.cmap = make(map[string]string)
 	b.ircNick = b.Config.IRC.Nick
+	b.names = make([]string, 0, 10000)
```

Comment:
This is not necessary for our use case, we don't need to allocate this much. We can just use the append function to grow dynamically.